docs/tests.md

```diff
@@ -97,16 +74,29 @@ Open Cypress, from there you can run the tests:
 npm run e2e
 ```
 
-### Run E2E tests against dev
+### Running E2E tests against dev or preprod env on CircleCI
+
+If the tests in the CI pipeline fail, you can try to reproduce by running the E2E tests in your local repo against dev or preprod env.
+
+You can run your local tests against dev env using:
+
 ```
-npx cypress open --env USERNAME=<username>,PASSWORD=<password>,USERNAME_SPO=<username_spo>,PASSWORD_SPO=<password_spo>,CRN=X098092,CRN2=X514364,ENV=dev --config-file e2e_tests/cypress.config.ts --config baseUrl=https://make-recall-decision-dev.hmpps.service.justice.gov.uk
+npx cypress open --env USERNAME=<username>,PASSWORD=<password>,USERNAME_SPO=<username_spo>,PASSWORD_SPO=<password_spo>,ENV=dev --config-file e2e_tests/cypress.config.ts --config baseUrl=https://make-recall-decision-dev.hmpps.service.justice.gov.uk
```

Comment:
Do we not need CRN's for the dev run?

Reply:
nope, the tests default to the correct ones